### PR TITLE
ref: adopt vscode's styling in beginner tips page

### PR DESCRIPTION
**Pull request by Eskibear**
- use vscode-webview-ui-toolkit

### CODE REVIEW

**Comment by Eskibear** [comment]
screenshots:

image

image

File: src/beginner-tips/assets/style.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    text-decoration: none;
+}
+
+h5 {
+    font-size: 1rem;
+    margin-top: 5px;
+    margin-bottom: 0;
+}
+
+@import "~@vscode/codicons/dist/codicon.css";
```
Comment (Eskibear) [inline src/beginner-tips/assets/style.scss:43]: probably need to update here after merging #800 ? @CsCherrYY

Comment (CsCherrYY) [reply src/beginner-tips/assets/style.scss:43]: Yes. Should replace `~` with relative path `../../../node_modules/`.

**Comment by Eskibear** [comment]
demo: 

image

**Comment by Eskibear** [comment]
@jdneo @CsCherrYY please help to take a look or play with the PR build, verify if it's working well.

File: src/beginner-tips/assets/tabs/QuickStartPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  ;
+  return (
+    <div className="tab-pane fade" id="quick-start-panel" role="tabpanel" aria-labelledby="quick-start-tab">
+      <p>
+        In this 1-minute tutorial, we'll show you how to create a quick-start Java program in VS Code.
+      </p>
+      <h2 className="font-weight-light">Setup the Workspace</h2>
+      <p>
+        VS Code Java works directly with <strong>folders</strong> that have source code. To setup the workspace, simply open a folder using File {">"}
+        {openFolderLink}
```
Comment (CsCherrYY) [inline src/beginner-tips/assets/tabs/QuickStartPanel.tsx:31]: suggest adding a new whitespace here. Currently it looks like:
image

File: src/beginner-tips/assets/tabs/QuickStartPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import React from 'react';
+
+const isMac: boolean = navigator.platform.toLowerCase().indexOf("darwin") === 0;
+
+export default function QuickStartPanel() {
+  const codeSample = (
+    <code>
```
Comment (CsCherrYY) [inline src/beginner-tips/assets/tabs/QuickStartPanel.tsx:10]: what's the font of this `code` tag?

Comment (Eskibear) [reply src/beginner-tips/assets/tabs/QuickStartPanel.tsx:10]: Seems a default one provided by VS Code's webview.

Comment (Eskibear) [reply src/beginner-tips/assets/tabs/QuickStartPanel.tsx:10]: it was overridden as `monospace` by agent. will update

**Comment by CsCherrYY** [comment]
just my own opinion, I just zoom in once than the default zoom in my PC (2k resolution), the font size of current version is a little small and a little hard to read many words in a page. The marketplace version has a gray background so it may look better. Can we consider increasing the font size?

**Comment by Eskibear** [comment]
> font-size: var(--type-ramp-base-font-size);

**Review by CsCherrYY — APPROVED** [review]

**Pull request merged**